docker-compose/swe-d2cc/config/swe-d2cc-export-application.yml

```diff
@@ -34,8 +34,9 @@ ftp:
   port: 21
   access-key: gridcapa
   secret-key: gridcapa
-  remote-relative-destination-directory: data/gridcapa/swe/d2cc/outputs
+  remote-relative-destination-directory: swe/d2cc/outputs # avec ça swe/d2cc/outputs le ftp directory /data/gridcapa/swe/d2cc/outputs mais ça ne peut pas se connecter ?
```

Comment:
if the comment is to be kept please put it in english

Reply:
I removed the comment